[2024-09-26T07:52:59.227+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Batch_api_to_csv_with_dynamic_dates.fetch_paginated_api_data manual__2024-09-26T07:52:58.429191+00:00 [queued]>
[2024-09-26T07:52:59.233+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Batch_api_to_csv_with_dynamic_dates.fetch_paginated_api_data manual__2024-09-26T07:52:58.429191+00:00 [queued]>
[2024-09-26T07:52:59.233+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 4
[2024-09-26T07:52:59.241+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): fetch_paginated_api_data> on 2024-09-26 07:52:58.429191+00:00
[2024-09-26T07:52:59.245+0000] {standard_task_runner.py:57} INFO - Started process 6931 to run task
[2024-09-26T07:52:59.248+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Batch_api_to_csv_with_dynamic_dates', 'fetch_paginated_api_data', 'manual__2024-09-26T07:52:58.429191+00:00', '--job-id', '77', '--raw', '--subdir', 'DAGS_FOLDER/common_authentication_data_pipeline.py', '--cfg-path', '/tmp/tmp7p4ujww4']
[2024-09-26T07:52:59.251+0000] {standard_task_runner.py:85} INFO - Job 77: Subtask fetch_paginated_api_data
[2024-09-26T07:52:59.276+0000] {task_command.py:410} INFO - Running <TaskInstance: Batch_api_to_csv_with_dynamic_dates.fetch_paginated_api_data manual__2024-09-26T07:52:58.429191+00:00 [running]> on host fb0671c6dbcd
[2024-09-26T07:52:59.609+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Batch_api_to_csv_with_dynamic_dates' AIRFLOW_CTX_TASK_ID='fetch_paginated_api_data' AIRFLOW_CTX_EXECUTION_DATE='2024-09-26T07:52:58.429191+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-26T07:52:58.429191+00:00'
[2024-09-26T07:52:59.612+0000] {logging_mixin.py:150} INFO - Start Date: 2024-09-25 01:00:00.000, End Date: 2024-09-26 01:00:00.000
[2024-09-26T07:52:59.765+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/common_authentication_data_pipeline.py", line 71, in fetch_data_and_save
    response.raise_for_status()
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 403 Client Error: Forbidden for url: http://34.124.138.144:8000/api/common/authentication
[2024-09-26T07:52:59.773+0000] {taskinstance.py:1350} INFO - Marking task as UP_FOR_RETRY. dag_id=Batch_api_to_csv_with_dynamic_dates, task_id=fetch_paginated_api_data, execution_date=20240926T075258, start_date=20240926T075259, end_date=20240926T075259
[2024-09-26T07:52:59.781+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 77 for task fetch_paginated_api_data (403 Client Error: Forbidden for url: http://34.124.138.144:8000/api/common/authentication; 6931)
[2024-09-26T07:52:59.828+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2024-09-26T07:52:59.843+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
